src/BizHawk.Client.Common/config/Config.cs

```diff
@@ -36,6 +36,10 @@ public class Config
 				new[] { CoreNames.Gambatte, CoreNames.Sameboy, CoreNames.GbHawk, CoreNames.SubGbHawk }),
 			(new[] { VSystemID.Raw.GBL },
 				new[] { CoreNames.GambatteLink, CoreNames.GBHawkLink, CoreNames.GBHawkLink3x, CoreNames.GBHawkLink4x }),
+			(new[] { VSystemID.Raw.GEN },
+				new[] { CoreNames.Gpgx, CoreNames.PicoDrive }),
+			(new[] { VSystemID.Raw.GG,VSystemID.Raw.SMS, VSystemID.Raw.SG },

```

Comment:
The first in the "key" array is used as the label, so SMS should probably be first.